pkg/framework/aries/api/vdr/vdr.go

```diff
@@ -24,17 +23,96 @@ const DIDCommServiceType = "did-communication"
 
 // Registry vdr registry.
 type Registry interface {
-	Resolve(did string, opts ...resolve.Option) (*did.DocResolution, error)
+	Resolve(did string, opts ...ResolveOption) (*did.DocResolution, error)
 	Store(doc *did.Doc) error
-	Create(method string, opts ...create.Option) (*did.DocResolution, error)
+	Create(method string, did *did.Doc, opts ...CreateOption) (*did.DocResolution, error)
 	Close() error
 }
 
 // VDR verifiable data registry interface.
 type VDR interface {
-	Read(did string, opts ...resolve.Option) (*did.DocResolution, error)
-	Store(doc *did.Doc, by *[]doc.ModifiedBy) error
-	Build(keyManager kms.KeyManager, opts ...create.Option) (*did.DocResolution, error)
+	Read(did string, opts ...ResolveOption) (*did.DocResolution, error)
+	Store(doc *did.Doc, by *[]ModifiedBy) error
```

Comment:
We should have a TODO to resolve the difference between Store and Create.

Reply:
I rename to create